src/conditions/base/timelock.ts

```diff
@@ -1,14 +1,19 @@
 import Joi from 'joi';
 
-import { Condition, makeReturnValueTest } from './condition';
+import { Condition } from './condition';
+import { makeReturnValueTest } from './schema';
 
 export class TimelockCondition extends Condition {
+  // TODO: This is the only condition that uses defaults, and also the only condition that uses `method` in order
+  //   to determine the schema. I.e. the only method that used `METHOD = 'timelock'` in `nucypher/nucypher`.
+  // TODO: Consider introducing a different field for this, e.g. `conditionType` or `type`. Use this field in a
+  //  condition factory.
```

Comment:
I think this `TimelockCondition` will change functionality to use on-chain time instead of system time. @KPrasch had some thoughts about doing that work.